### ISSUE TITLE
Stahování filmů s původním názvem a titulky

### ISSUE THREAD

**Issue by tprochazka** [bbaronSVK/plugin.video.stream-cinema#20]
V současné verzi se společně s filemem vůbec nestáhnou titulky a název je změněm, přitom API poskytuje správný originální název. 

Obojí jsem opravil v tomto pull requestu
https://github.com/bbaronSVK/plugin.video.stream-cinema/pull/15

**Comment by tprochazka** [comment]
Fakt nechápu, proč je s tímto takový problém, evidentně je to chyba, celá funkce je kvůli tomu úplně nepoužitelná :-(

**Comment by bbaronSVK** [comment]
problem je v tom, originalnom nazve, ktory uz mimochodom davnejsie v API nechodi. Su tam ale zachovane udaje o zdroji, pripadne grupe ak sa nachadzaju aj v originali. Ide o to, ze su tu urcity ludia, ktory tuto informaciu zneuzivaju. Musel som najprv poriesit tento problem.

**Comment by bbaronSVK** [comment]
commitnute v 1.3

**Comment by tprochazka** [comment]
No, všiml jsem si, že už z toho API chodí nesmysly :-( Nechápu, ja kto může někdo zneužívat, k tomu názvu se dá stejně nadále dopátrat přímo přes webshare (pokud už to tedy není změněno při uploadu). Ale někdy nesedí nebo chybí titulky a bez správného názvu je nemožné, najít ty správné, i když vznikne přečasování pro tu verzi. Často se stává, že pro jeden film je tam třeba 10 zdrojů a titulky sedí jen několik z nich.
